### ISSUE TITLE
Weird map issue on OS X platform

### ISSUE THREAD

**Issue by nntoan** [simsulla/nwomod#855]
This issue only appear on my Macbook, my iMac doesn't have this.
I already tried:

- [x] Copy map cache from iMac to Macbook

- [x] Clear map cache from both `/com.vpltd.Victoria2HeartOfDarkness-MAS/GameDocuments/Paradox Interactive/Victoria II/map` and `/Applications/Victoria 2 - Heart Of Darkness.app/Contents/Resources/Data/map/`

- [x] Remove water.dds and its companions

- [x] Reinstall NWO mod 1000 times

 But unfortunately, nothing changes. The issue still there. I have no idea what it is, please check the screenshot below.

![screenshot 2016-12-19 15 06 57](https://cloud.githubusercontent.com/assets/3891429/21308675/941fbd82-c60d-11e6-9a57-8e0050013a7f.png)

**Comment by nntoan** [comment]
In case you need logs.
[logs.zip](https://github.com/simsulla/nwomod/files/660733/logs.zip)

**Comment by simsulla** [comment]
do you see this with the official 1.11 [download install](http://simsulla.org/#download) as well or only the latest from the nwomod-master.zip

**Comment by nntoan** [comment]
@simsulla Yes, I got this problem with the official download (on homepage) first, before I tried with GitHub.

**Comment by simsulla** [comment]
Ok, so the iMac works fine, but Macbook has problems. It seems to be graphics card (settings) related. Well see if I can get hold of a mac and investigate,

**Comment by simsulla** [comment]
Confirmed the issue on my mac mini. The map overlay is no good.... At the moment no idea why.

**Comment by simsulla** [comment]
I am not able to resolve this issue. My guess is that some of the map graphics is of wrong format in NWO, and that this causes the "dark" map. But at the moment unable to detect which map it is. Will move this issue out of 2.00 and brand it as a core bug, though this strictly is not the case (I think!).

**Comment by nntoan** [comment]
Sad :(
